insilico_panels/KK/neurometabola_sjukdomar.v1.0/neurometabola_sjukdomar.v1.0

```diff
@@ -93,3 +93,6 @@ SLC17A5
 SMPD1
 SUMF1
 TPP1
+DHCR7
+CYP27A1
+
```

Comment:
I'm thinking that this should cause the panel to become version 2.0? So we can keep track on how this differs from version 1.0

Reply:
Yes, I agree. Maybe theer could be a README about the changes to genelists and bedfiles (v1 and v2 and so on) in the repo insilico_wgs/insilico_panels/ ?

Reply:
Or we can have a README in each panel subfolder, listing which changes were made, why and by whom for each panel.

Reply:
Yes, maybe that would be more clear.